src/gridiron/Node.hpp

```diff
@@ -4,8 +4,10 @@
 #include <htmlcxx2/htmlcxx2_html.hpp>
 #include <htmlcxx2/kp_tree.hh>
 
+#include <gridiron/properties.hpp>
+
 namespace GridIron {
-    class Node : public htmlcxx2::HTML::Node {
+    class Node : std::enable_shared_from_this<GridIron::Node>, public htmlcxx2::HTML::Node {
```

Comment:
You missed the 'public' here!  You must always have enable_shared_from_this as a public base, or the shared_ptr won't find it and it won't work.

Reply:
Thanks for the catch on this too!